cli/azd/pkg/apphost/generate.go

```diff
@@ -274,6 +286,60 @@ func newInfraGenerator() *infraGenerator {
 	}
 }
 
+func evaluateForOutputs(value string) (map[string]genOutputParameter, error) {
+	outputs := make(map[string]genOutputParameter)
+
+	re := regexp.MustCompile(`\{[a-zA-Z0-9\-]+\.(outputs|secretOutputs)\.[a-zA-Z0-9\-]+\}`)
+	matches := re.FindAllString(value, -1)
+	for _, match := range matches {
+		noBrackets := strings.TrimRight(strings.TrimLeft(match, "{"), "}")
+		parts := strings.Split(noBrackets, ".")
+		name := fmt.Sprintf("%s_%s", strings.ToUpper(parts[0]), strings.ToUpper(parts[2]))
+		outputType := "outputs"
+		if strings.Contains(match, "secretOutputs") {
+			outputType = "secretOutputs"
+		}
+		outputs[name] = genOutputParameter{
+			Type:  "string",
+			Value: fmt.Sprintf("%s.%s.%s", parts[0], outputType, parts[2]),
```

Comment:
Just to help me understand - what is the difference between using `outputType` here and using `parts[1]`.  Are there cases where `outputType` is `secretOutputs` yet `parts[1]` would be `outputs`.  Trying to understand if this ends up just being the value that was passed in to `evaluateForOutputs` but without the {}`s?

Reply:
parts[1] shoudl work!  Thank you for noticing <3